docs/openapi.yaml

```diff
@@ -48,7 +48,19 @@ tags:
   - name: Index Querying
     description: |
       Get data from the AR.IO Gateway index using GQL
+  - name: Admin
+    description: |
+      Access several password protected features and functions specific to your AR.IO Gateway.  
+
+
+      Input your ADMIN_API_KEY using the green "Authorize" button at the top of this page, or by clicking on the "lock" icon next to any password protected endpoint.
 components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: userPassword
```

Comment:
I don't think `userPassword` is quite right here. It should probably be something like `apiToken`.